pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>life.qbic</groupId>
 		<artifactId>parent-pom</artifactId>
-		<version>1.5.1</version>
+		<version>2.2.0</version>
 	</parent>
 	<artifactId>data-model-lib</artifactId>
 	<version>1.7.0-SNAPSHOT</version>
```

Comment:
this will be done on the master branch

Reply:
Alternatively, you could create a `prepare-release` branch so that it is not the `development` branch and already change it there. So you don't forget to do that.